src/components/SuperAdminScreen/SuperAdminScreen.tsx

```diff
@@ -4,7 +4,6 @@ import Button from 'react-bootstrap/Button';
 import { useTranslation } from 'react-i18next';
 import { Outlet, useLocation } from 'react-router-dom';
 import styles from './SuperAdminScreen.module.css';
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`. 
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296